field/TinyLang/Field/Printer.hs

```diff
@@ -60,3 +60,6 @@ toStringNoIDs = toString NoIDs
 -- | Convert an Expr to a String, including Unique IDs in variable names
 toStringWithIDs :: Show f => Expr f a -> String
 toStringWithIDs = toString WithIDs
+
+tos :: (Show f) => SomeUniExpr f -> String
+tos (SomeUniExpr _ e) = toStringNoIDs e
```

Comment:
Sorry!  I stuck that in for experiments and then forgot to make it more sensible.

Reply:
I've now made this someExprToString and exposed the `WithIDs | NoIDs` type in the module interface instead of having separate `exprToStringNoIDs` and `exprToStringWithIDs` functions.